src/AutoBuild.jl

```diff
@@ -982,15 +993,15 @@ function rebuild_jll_package(name::String, build_version::VersionNumber, sources
             )
         end
 
-        # If `from_scartch` is set (the default) we clear out any old crusty code
+        # If `from_scratch` is set (the default) we clear out any old crusty code
         # before generating our new, pristine, JLL package within it.  :)
         if from_scratch
             rm(joinpath(code_dir, "src"); recursive=true, force=true)
             rm(joinpath(code_dir, "Artifacts.toml"); force=true)
         end
 
         # Finally, generate the full JLL package
-        build_jll_package(name, build_version, sources, code_dir, build_output_meta, dependencies, upload_prefix; verbose=verbose, lazy_artifacts=lazy_artifacts)
+        build_jll_package(name, build_version, sources, code_dir, build_output_meta, dependencies, upload_prefix; verbose=verbose, lazy_artifacts=lazy_artifacts, init_block=init_block)
```

Comment:
Perhaps this line should be wrapped? Are you using `JuliaFormatter` or something like that?

Reply:
No, but I usually try to wrap things at about 90 characters.  So yes please, feel free to wrap it.

Reply:
Done. (Lots of lines > 90 in the file, but ... *shrug* ;-)